.gitignore

```diff
@@ -5,3 +5,4 @@ coverage
 *.sw?
 .DS_Store
 runtime.json
+.idea
```

Comment:
yep, it's a hidden folder where jetbrains IDE generate all sort of project files